jdi-light/src/main/java/com/epam/jdi/light/elements/composite/WebPage.java

```diff
@@ -108,27 +151,31 @@ public static String getTitle() {
     void setUrl(String uri) {
         setUrl(uri, "", CONTAINS);
     }
+
     void setUrl(String uri, String template, CheckTypes validate) {
         url = uri;
         checkUrl = template;
-        checkUrlType = validate;
+      //  checkUrlType = validate;
         if (isBlank(template)) {
             if (validate != MATCH)
                 checkUrl = uri;
             else throw exception("In order to validate MATCH for page '%s', please specify 'template' in @Url",
                     getName());
-        } else if (validate == null) checkUrlType = MATCH;
+        } else if (validate == null) ;//checkUrlType = MATCH;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [An empty statement (semicolon) not part of a loop](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5327794)